packages/react-dom/src/events/accumulateTwoPhaseListeners.js

```diff
@@ -127,6 +135,30 @@ export default function accumulateTwoPhaseListeners(
             dispatchInstances.push(node);
           }
         }
+      } else if (enableScopeAPI && node.tag === ScopeComponent) {
+        const reactScope = node.stateNode.methods;
+        const eventTypeMap = reactScopeListenerStore.get(reactScope);
+        if (eventTypeMap !== undefined) {
+          const type = ((event.type: any): DOMTopLevelEventType);
+          const listeners = eventTypeMap.get(type);
+          if (listeners !== undefined) {
+            const captureListeners = Array.from(listeners.captured);
```

Comment:
btw I noticed we use this in a few places right now, so we should probably add Array.from to this page https://reactjs.org/docs/javascript-environment-requirements.html

Reply:
Good point, although we don't need to add it till v17.